test/src/test/java/org/corfudb/runtime/checkpoint/CheckpointTest.java

```diff
@@ -194,7 +194,7 @@ public void periodicCkpointTrimTest() throws Exception {
         scheduleConcurrently(1, ignored_task_num -> {
             for (int i = 0; i < mapSize; i++) {
                 m2A.put(String.valueOf(i), (long)i);
-                m2B.put(String.valueOf(i), (long)0);
+                m2B.put(String.valueOf(i), (long)i);
```

Comment:
By fix, I mean fix it to do what you meant. I just fixed it since it seemed to be incorrect